snapm/manager/_manager.py

```
@@ -499,6 +503,30 @@ def wrapper(*args, **kwargs):
+    it if necessary.
+    """
+    mountsdir = _SNAPM_MOUNTS_DIR
+    makedirs(mountsdir, mode=_SNAPM_MOUNTS_DIR_MODE, exist_ok=True)
+    try:
+        os.chmod(mountsdir, _SNAPM_MOUNTS_DIR_MODE)
+    except OSError as err:
+        st = os.stat(mountsdir)
+        if (st.st_mode & 0o777) != _SNAPM_MOUNTS_DIR_MODE:
+            raise SnapmSystemError(
+                f"Lock directory {mountsdir} has insecure permissions"
+            ) from err
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)